terracumber_config/tf_files/SUSEManager-4.3-build-validation-NUE.tf

```diff
@@ -804,6 +804,25 @@ module "sles15sp5s390-minion" {
   ssh_key_path            = "./salt/controller/id_rsa.pub"
 }
 
+module "salt-migration-minion" {
+  source = "./modules/minion"
+  base_configuration = module.base.configuration
+  name               = "min-salt-migration"
+  product_version    = "4.3-released"
+  image              = "sles15sp5o"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:cf"
```

Comment:
are you sure?
```
infrastructure/srv/salt/dhcpd-server> grep -Ri aa:b2:92:42:00:cf
infrastructure/srv/salt/dhcpd-server> 
```

Reply:
```
infrastructure/srv/salt> grep -Ri aa:b2:92:42:00:cf
dhcpd-server/mgr_prv_suse_net/dhcpd.conf:host suma-bv-43-min-salt-migration { hardware ethernet aa:b2:92:42:00:cf; fixed-address 10.84.221.207; option host-name "suma-bv-43-min-salt-migration"; }
dhcpd-server/mgr_suse_de/dhcpd.conf:host suma-bv-43-min-salt-migration   { hardware ethernet aa:b2:92:42:00:cf; fixed-address 10.145.209.207; option host-name "suma-bv-43-min-salt-migration";  }
```

I've got them. Maybe you did not pull the last changes ? ( same for the other comments)

Reply:
Right, i forgot the git pull